backend/business/core/collectors/github/fetchers/repos_info_fetcher.go

```diff
@@ -54,6 +56,11 @@ type repoInfo struct {
 		}
 		OpenIssues struct {
 			TotalCount githubv4.Int
+			Nodes      []struct {
+				Issue struct {
+					CreatedAt githubv4.DateTime
+				}
+			}
```

Comment:
```suggestion
			Nodes      []struct {
				CreatedAt githubv4.DateTime
			}
```

Reply:
This was just one level of indirection away from being able to run 🏃 

`Issues` isn't a field on `Nodes` so we can't access it. But once we remove it then we're good to go